geniza/corpus/models.py

```diff
@@ -1091,15 +1093,15 @@ class TextBlock(models.Model):
             + "Uncheck this box if you are uncertain of a potential join."
         ),
     )
-    RECTO = "r"
-    VERSO = "v"
-    RECTO_VERSO = "rv"
-    RECTO_VERSO_CHOICES = [
-        (RECTO, "recto"),
-        (VERSO, "verso"),
-        (RECTO_VERSO, "recto and verso"),
-    ]
-    side = models.CharField(blank=True, max_length=5, choices=RECTO_VERSO_CHOICES)
+    RECTO = _("recto")
+    VERSO = _("verso")
+    RECTO_VERSO = _("recto and verso")
```

Comment:
I don't think we need to translate these — AFAIK they won't be displayed on the public site, only in the admin and in the data downloads.